src/kit/Tooltip.tsx

```diff
@@ -44,7 +59,9 @@ const Tooltip: React.FC<Props> = ({
       open={open}
       overlayClassName={className}
       placement={placement}
+      ref={tootipContainer}
```

Comment:
are you sure this ref goes to the tooltip trigger?

Reply:
html elements dont have a `props` attribute. Can you test this to make sure it works?